plugins/mobile_channel_header_plug/mobile_channel_header_plug.tsx

```diff
@@ -0,0 +1,195 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {injectIntl, IntlShape} from 'react-intl';
+
+import {sendEphemeralPost} from 'actions/global_actions';
+import {AppCallResponseTypes, AppCallTypes} from 'mattermost-redux/constants/apps';
+import {ActionResult} from 'mattermost-redux/types/actions';
+import {AppBinding, AppCallRequest, AppCallResponse, AppCallType} from 'mattermost-redux/types/apps';
+import {Channel, ChannelMembership} from 'mattermost-redux/types/channels';
+import {Theme} from 'mattermost-redux/types/preferences';
+
+import {PluginComponent} from 'types/store/plugins';
+import {createCallContext, createCallRequest} from 'utils/apps';
+
+type Props = {
+
+    /*
+         * Components or actions to add as channel header buttons
+         */
```

Comment:
The comments here seem to be misaligned compared to the javascript file this came from